src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/ExcelCreationServiceForReport.java

```diff
@@ -0,0 +1,240 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.service.excel;
+
+import com.google.common.base.Strings;
+import io.netty.util.internal.StringUtil;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.poi.ss.usermodel.BorderStyle;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellStyle;
+import org.apache.poi.ss.usermodel.FillPatternType;
+import org.apache.poi.ss.usermodel.Font;
+import org.apache.poi.ss.usermodel.HorizontalAlignment;
+import org.apache.poi.ss.usermodel.IndexedColors;
+import org.apache.poi.ss.usermodel.VerticalAlignment;
+import org.apache.poi.ss.util.CellRangeAddress;
+import org.apache.poi.xssf.usermodel.XSSFRow;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.ecm.common.helpers.UtilHelper;
+import uk.gov.hmcts.ethos.replacement.docmosis.reports.ReportException;
+import uk.gov.hmcts.ethos.replacement.docmosis.reports.claimsbyhearingvenue.ClaimsByHearingVenueReportData;
+import uk.gov.hmcts.ethos.replacement.docmosis.reports.claimsbyhearingvenue.ClaimsByHearingVenueReportDetail;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Slf4j
+@Service("excelCreationServiceForReport")
+public class ExcelCreationServiceForReport {
```

Comment:
The name of the class suggests it could be used for any report but it is specific to one. You should name it more appropriately

Reply:
And probably change the package. The service/excel package doesn't seem quite right either. 
service/reports/claimsbyhearingvenue?

Reply:
Renamed to show the specific report type.